gradle.properties

```diff
@@ -22,7 +22,6 @@ kotlin.code.style=official
 # resources declared in the library itself and none from the library's dependencies,
 # thereby reducing the size of the R class for that library
 android.nonTransitiveRClass=true
-MAPBOX_DOWNLOADS_TOKEN=sk.eyJ1IjoibXVsdGltYXR1bSIsImEiOiJjbDMwNWZsODgxN2dyM2Nqenk2dGt0NzFtIn0.Wgj-43xPKrBmAP7TkzFGBQ
```

Comment:
Why did you remove this token ?

Reply:
As it is used in the project this line is useless.